raster/r.proj/testsuite/test_rproj.py

```diff
@@ -0,0 +1,179 @@
+"""
+Name:       r.texture test
+Purpose:    Tests r.texture and its flags/options.
+
+Author:     Sunveer Singh, Google Code-in 2017
+            Chung-Yuan Liang, modified in 2024
+Copyright:  (C) 2017 by Sunveer Singh and the GRASS Development Team
+Licence:    This program is free software under the GNU General Public
+            License (>=v2). Read the file COPYING that comes with GRASS
+            for details.
+"""
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.checkers import text_to_keyvalue
+from grass.gunittest.gmodules import call_module
+import os
+
+raster_info = """north=35.8096296297222
+south=35.6874074075
+east=-78.608
+west=-78.7746666666667
+nsres=0.000740740740740727
+ewres=0.000666666666666686
+rows=165
+cols=250
+cells=41250"""
+
+
+class TestRasterreport(TestCase):
+    input = "elevation"
+
+    @classmethod
+    def setUpClass(cls):
+        cls.runModule("g.proj", project="nc_latlong", epsg="4326", flags="c")
+        cls.runModule("g.mapset", mapset="PERMANENT", project="nc_latlong")
+
+    @classmethod
+    def tearDownClass(cls):
+        dbase = call_module("g.gisenv", get="GISDBASE")
+        os.system(f"rm -rf {dbase}/nc_latlong")
```

Comment:
I cannot find a way to remove the project with another CRS that I created, so I use the shell comment `rm -rf`. However, this disobeys the CI rule. If anyone has any suggestions for this, please let me know. Thanks.